### DISCUSSION TITLE
Umlauts in document path

### DISCUSSION THREAD

**Discussion by vlad12244** [pymupdf/PyMuPDF#2551] [Looking for help] [answered]
I am using MuPDF in my C# project, and it seems that MuPDF can't read the file path if it contains non-English characters like umlauts (äöüõ). Is there a way to bypass this issue? One possible solution I thought of is to temporarily copy the file to a different location, process it with MuPDF, and then copy it back. However, I'm aware that this might not be the best approach. The error message I am receiving is: MuPDFCore.MuPDFException: 'Cannot open document'.

**Comment by JorjMcKie** [comment]
It seems you have posted on the wrong repo - this repo us for PyMuPDF, the Python bindings for MuPDF.
Maybe you should make a post in MuPDF 's public Discord channel and ask for help there:
https://discord.com/channels/770681584617652264/770685907279282196

**Comment by JorjMcKie** [comment, marked as answer]
As far as PyMuPDF is concerned, you are able to use UTF-8 encoded file names. You can also read a file to memory and then open this stream of bytes like any document.

**Reply by vlad12244** [reply]
Thank you for your answer.  I will try that in my code.
